.github/actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -185,111 +187,287 @@ async function getCheckComment( octokit, owner, repo, number ) {
 }
 
 /**
- * Checks the contents of a PR description.
+ * Compose a list item with appropriate status check and passed message
+ *
+ * @param {boolean} isFailure - Boolean condition to determine if check failed.
+ * @param {string} checkMessage - Sentence describing successful check.
+ * @param {string} severity - Optional. Check severity. Could be one of `error`, `warning`, `notice`
+ *
+ * @returns {string} - List item with status emoji and a sentence describing check.
+ */
+function statusEntry( isFailure, checkMessage, severity = 'error' ) {
+	const severityMap = {
+		error: ':red_circle:',
+		warning: ':warning:',
+		notice: ':spiral_notepad:',
+		ok: ':white_check_mark:',
+	};
+	const status = isFailure ? severityMap[ severity ] : severityMap.ok;
+	return `
+- ${ status } ${ checkMessage }<br>`;
+}
+
+/**
+ * Returns list of projects with missing changelog entries
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Array} - list of affected projects without changelog entry
+ */
+async function getChangelogEntries( octokit, owner, repo, number ) {
+	const files = await getFiles( octokit, owner, repo, number );
+	const affectedProjects = getAffectedChangeloggerProjects( files );
+	debug( `check-description: affected changelogger projects: ${ affectedProjects }` );
+
+	return affectedProjects.reduce( ( acc, project ) => {
+		const found = files.find( file => file.includes( project ) && file.includes( '/changelog/' ) );
```

Comment:
Note the directory name could theoretically be changed by setting `.extra.changelogger.changes-dir` in composer.json. I don't know why we would, but it's possible.

Reply:
Do you think it worth addressing?

Reply:
:shrug: If we don't address it now, it'll probably not get addressed until someone trips over it after setting `.extra.changelogger.changes-dir`. But that may never happen.

I'd at least stick a comment in pointing it out:
```suggestion
		// todo: Check `.extra.changelogger.changes-dir` in composer.json instead of assuming the default "changelog".
		const found = files.find( file => file.includes( project ) && file.includes( '/changelog/' ) );
```

Reply:
addressed in 8619f26